setup_undercloud.yml

```diff
@@ -22,6 +23,7 @@
         hypervisor_host: "{{ undercloud_hostname }}"
         hypervisor_password: "{{ ansible_ssh_pass }}"
       include_tasks: tasks/install_os.yml
+      when: osp_release is version('17.1', '<')
```

Comment:
This need to be removed once the 9.1 image is available right? if so, add a TODO comment

Reply:
@masco I think RHEL 9.1 is not yet the default in passed_phase2 for OSP 17.1.
@asyedham What do you think? Should I change to 9.0? In that case we don't need to skip this now.

Reply:
Yeah we can do that @masco what are your thoughts?

Reply:
The latest passed_phase2 puddle of OSP 17.1 uses RHEL 9.1.